images/base/Dockerfile

```diff
@@ -144,6 +145,7 @@ RUN echo "Installing containerd ..." \
     && chmod 755 /usr/local/sbin/runc \
     && containerd --version \
     && runc --version \
+    && ctr oci spec |jq '.mounts[.mounts | length] |= . + {"destination": "/sys/class/dmi/id/product_uuid", "source": "/proc/sys/kernel/random/uuid", "options": ["bind"]}'> /etc/containerd/cri-base.json \
```

Comment:
Mount should be read only?
Why wouldn't we mount the file from the kind node instead of a random file per container?

Reply:
Do we have to mimic https://github.com/kubernetes-sigs/kind/blob/main/images/base/files/usr/local/bin/entrypoint#L280-L289 too ?